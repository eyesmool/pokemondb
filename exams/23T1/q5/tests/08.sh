python3 ./q5.py 'St Leonards'
