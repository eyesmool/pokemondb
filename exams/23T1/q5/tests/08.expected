St Leonards branch (2041) holds
- account 606122 owned by Andrew Lin from Annandale with $200
- account 608457 owned by Jiahui Goh from Ryde with $0
- account 611967 owned by John Kirby-Lewis from Birchgrove with $400
- account 612853 owned by Ling Poon from Peakhurst with $1100
- account 624154 owned by Lynn Qu from Minchinbury with $0
- account 626899 owned by Kyung Lee from Wollstonecraft with $400
- account 627147 owned by Wendy Todd from Pymble with $400
- account 627631 owned by Kevin Sng from Summer Hill with $900
- account 633587 owned by Alexander Mitchell from Church Point with $1200
- account 634085 owned by Yongheng Zhang from Cremorne with $10000
- account 635644 owned by Patricia Outten from Blair Athol with $500
- account 636416 owned by Qin Hua from Leppington with $1000
- account 641456 owned by Charnwit Thongplubplanis from Bilgola Beach with $400
- account 644969 owned by Lin Jiang from Macarthur with $700
- account 645874 owned by Kristopher Murphy from Pymble with $1200
- account 650559 owned by Ying Ng from Peakhurst with $1800
- account 653114 owned by Ling Song from Kogarah with $400
- account 653538 owned by Martin Chan from Narrabeen with $1500
Assets: $27600
Discrepancy between assets and sum of account balances
